src/main.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function(){
+
+  var game = new Game();
+  drawBoard( game.numberOfRows, game.numberOfColumns);
+  paintCells( game.board );
+
+  setInterval( function(){
+    game.iterate();
+    paintCells( game.board );
+  }, 500);
+
+});
+
+var paintCells= function(board){
+  
+  $.each(board.coordinates, function(cell){
+    point = cell.split(':');
+   
+    var rowId = 'row-' + point[0]; 
+    var colId = 'column-' + point[1]; 
+    $('#'+rowId + '>#' +colId).toggleClass('alive', board.coordinates[cell].isAlive  ); 
+  });  
+
+}
+
+var drawBoard = function( rows, columns){
```

Comment:
In the next iteration, try displaying it on a canvas element instead of in a table.